Wikipedia/UI-V5/UIView+WMFShadow.m

```diff
@@ -11,16 +11,11 @@
 @implementation UIView (WMFShadow)
 
 - (void)wmf_setupShadow {
-    self.clipsToBounds       = NO;
+    self.layer.masksToBounds = NO;
```

Comment:
@bgerstle 
Not a proxy iirc as a view can have many layers. Switching this line to the layer's maskToBounds means we can now apply shadows to things like UITableView, which can look wonky if you turn off clipsToBounds.